press/press/doctype/team/team.py

```diff
@@ -607,12 +607,14 @@ def update_billing_details(self, billing_details):
 				{"link_doctype": self.doctype, "link_name": self.name, "link_title": self.name},
 			)
 
+		pincode = billing_details.get("postal_code", "").strip().replace(" ", "")
+
 		address_doc.update(
 			{
 				"address_line1": billing_details.address,
 				"city": billing_details.city,
 				"state": billing_details.state,
-				"pincode": (billing_details.postal_code).strip().replace(" ", ""),
+				"pincode": pincode,
```

Comment:
no need to create another variable for this since it won't be used again.
```suggestion
				"pincode": billing_details.get("postal_code", "").strip().replace(" ", ""),
```